editor-lsp/src/main/java/io/github/rosemoe/sora/lsp/events/document/DocumentChangeEvent.kt

```diff
@@ -81,13 +81,9 @@ class DocumentChangeEvent : AsyncEventListener() {
         editor: LspEditor,
         data: ContentChangeEvent
     ): List<TextDocumentContentChangeEvent> {
-        val text = data.changedText.toString()
+        val text = data.editor.text.toString()
```

Comment:
This does not comply with the LSP specification; what is required here is text for an incremental change. Please clarify why there is a need to modify this part, and if possible, provide a project that can reproduce the related bug. I will look into it.